cpp/python/pystrings.cpp

```diff
@@ -135,6 +136,52 @@ bool parse_args( const char* fn, PyObject* pyargs, const char* pyfmt, ... )
     return rtn;
 }
 
+static std::basic_string<char> ConvertCudaIpcMemHandler (cudaIpcMemHandle_t ipc_memhandle) {
```

Comment:
Could you match the style of the other methods here? Braces and spaces.

Reply:
Done.